src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -233,16 +233,11 @@ public void asyncGetCube(int cubeX, int cubeY, int cubeZ, @Nonnull Requirement r
 		}
 
 		if (cube == null) {
-			asyncGetColumn(cubeX, cubeZ, req, col -> {
-				if (col == null) {
-					callback.accept(null);
-					return;
-				}
-				AsyncWorldIOExecutor.queueCubeLoad(worldObj, cubeIO, col, cubeY, loaded -> {
-					onCubeLoaded(loaded, col);
-					loaded = postCubeLoadAttempt(cubeX, cubeY, cubeZ, loaded, col, req);
-					callback.accept(loaded);
-				});
+			AsyncWorldIOExecutor.queueCubeLoad(worldServer, cubeIO, this, cubeX, cubeY, cubeZ, loaded -> {
+				Column col = getLoadedChunk(cubeX, cubeZ);
```

Comment:
This may be null - are we just relying on never calling this before loading the column?